pages/_document.js

```diff
@@ -16,8 +16,13 @@ const getCssAndReset = () => {
 
 export default class Document extends NextDocument {
   static async getInitialProps(ctx) {
-    const initialProps = await NextDocument.getInitialProps(ctx);
-    return { ...initialProps };
+    let initialProps = { html: "<></>" };
```

Comment:
is expecting to receive initialProps for components that we have doc pages for even though they haven't been built yet.

Reply:
Great!